### ISSUE TITLE
Fatal: worker: not found

### ISSUE THREAD

**Issue by jensim** [sourcegraph/sourcegraph#22376]
- **Sourcegraph version:** 3.29.0 OSS single docker instance setup
- **Platform information:** 

#### Build setup:
(Running in github actions)

```
#!/bin/bash

set -xe

cd "$(git rev-parse --show-toplevel)"

[ -n "$docker_username" ]
[ -n "$docker_password" ]
[ -n "$docker_repo" ]

docker login --username "$docker_username" --password "$docker_password"

curl https://api.github.com/repos/sourcegraph/sourcegraph/tags -o /tmp/sourcegraph_tags.json
tag="$(jq -r '.[] |.name' /tmp/sourcegraph_tags.json | grep -E '^v[0-9]+\.[0-9]+\.[0-9]+$' | head -n 1)"

curl "https://hub.docker.com/v2/repositories/${docker_username}/${docker_repo}/tags" -o /tmp/docker_tags.json

if jq -r '.results[] | .name' /tmp/docker_tags.json | grep -E "^${tag}$"; then
  echo "======================================
=  Tag '$tag' already in docker hub  =
======================================"
  exit 0
fi

export IMAGE="$docker_username/$docker_repo"
export VERSION="$tag-oss"

cd sourcegraph

git fetch origin refs/tags/$tag:refs/tags/$tag
git checkout "tags/$tag" -b "$tag-release-branch-$docker_username"

if [ 'Linux' == "$(uname -s)" ]; then
  sudo apt-get install musl-tools
fi

cd cmd/server
./pre-build.sh
./build.sh

docker tag "${docker_username}/${docker_repo}:latest" "${docker_username}/${docker_repo}:${tag}"
docker tag "${docker_username}/${docker_repo}:latest" "${docker_username}/${docker_repo}:${VERSION}"

docker push --all-tags "${docker_username}/${docker_repo}"

echo '===========
=  Done!  =
==========='

```

#### Steps to reproduce:

I've built sourcegraph oss so many times, I have now a release pipeline for it on github, publishing to my private dockerhub repo
Latest release, 3.29.0 doesn't start though.

```
services:
  sourcegraph_oss:
    image: $IMAGE:$VERSION
    networks:
      - sg_network
    volumes:
      - .sourcegraph/config:/etc/sourcegraph
      - .sourcegraph/data:/var/opt/sourcegraph
      - .sourcegraph/site-conf:/root
      - .sourcegraph/ca-cert:/usr/local/share/ca-certificates/
```

1. docker-compose -f docker-compose.yml up -d
2. <CRASH>

#### Expected behavior:

2. container start in a second of two

#### Actual behavior:

2. worker is unable to start
```
02:57:14 worker | /bin/sh: exec: line 1: worker: not found
02:57:14 worker | Terminating worker
02:57:14 repo-updater | Terminating repo-updater
02:57:14 frontend | Terminating frontend
02:57:14 query-runner | Terminating query-runner
02:57:14 gitserver | Terminating gitserver
02:57:14 github-proxy | Terminating github-proxy
02:57:14 searcher | Terminating searcher
02:57:14 symbols | Terminating symbols
02:57:14 jaeger | Terminating jaeger
02:57:14 postgres | Terminating postgres
02:57:14 zoekt-webserver | Terminating zoekt-webserver
02:57:14 grafana | Terminating grafana
02:57:14 prometheus | Terminating prometheus
02:57:14 postgres_exporter | Terminating postgres_exporter
02:57:14 nginx | Terminating nginx
02:57:14 syntect_server | Terminating syntect_server
02:57:14 zoekt-indexserver | Terminating zoekt-indexserver
02:57:14 minio | Terminating minio
02:57:14 redis-cache | Terminating redis-cache
02:57:14 redis-store | Terminating redis-store
```

**Comment by panda87** [comment]
Experiencing the same issue - @jensim do you have any quick fix for that?

**Comment by jensim** [comment]
@panda87 , im afraid not ... 
I thinkn the oss version of the worker container setup has not been tested, might be that it was forgotten 🤠

**Comment by aflag** [comment]
fyi, 3.28.0 is the last known good version

**Comment by efritz** [comment]
Sorry about this; I added the worker to the enterprise build but not the OSS default list. Should be fixed by https://github.com/sourcegraph/sourcegraph/pull/22621.

**Comment by panda87** [comment]
@efritz should we see that only on the next version? or can we recreate the current oss version?

**Comment by efritz** [comment]
The next release will contain the fix. 3.29 currently remains broken, but if you are building off of main/insiders then those should be fixed as well.

**Comment by panda87** [comment]
My point is that when v3.30 will be released if we can upgrade 3.28 -> 3.30 w/o jumping into 3.29?
